### PR TITLE
Register user

### PR DESCRIPTION
**Pull request by bartek3xe**
### Goal
Register user

### Proof of the implementation
Screenshot from 2023-11-12 20-09-28

### Trello
https://trello.com/c/lzOKox74

### CODE REVIEW

**Comment by bartek3xe** [comment]
To do: Hash password, allow polish characters in database, add role

File: backend/config/packages/twig.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+twig:
```
Comment (JozefPotaczek) [inline backend/config/packages/twig.yaml:1]: I don't think we want to use twig in this project so twig bundle is unnecessary

File: backend/config/bundles.php [hunk 2]
```diff
@@ -6,4 +6,7 @@
     Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle::class => ['all' => true],
     Symfony\Bundle\MakerBundle\MakerBundle::class => ['dev' => true],
     Nelmio\CorsBundle\NelmioCorsBundle::class => ['all' => true],
+    Symfony\Bundle\SecurityBundle\SecurityBundle::class => ['all' => true],
+    Symfony\Bundle\TwigBundle\TwigBundle::class => ['all' => true],
```
Comment (JozefPotaczek) [inline backend/config/bundles.php:10]: Unnecessary twig reference

File: backend/composer.lock
Comment (JozefPotaczek) [inline backend/composer.lock:1]: Unnecessary twig bundle

**Review by JozefPotaczek — CHANGES_REQUESTED** [review]
Remove unnecessary twig bundle

File: backend/config/packages/twig.yaml (hunk 1 above)
Comment (bartek3xe) [reply backend/config/packages/twig.yaml:1]: We need a twig bundle to use `make` command in the future.

**Review by JozefPotaczek — APPROVED** [review]

**Review by KiviGibson — APPROVED** [review]

**Pull request merged**